webapp/graphite/render/evaluator.py

```diff
@@ -58,7 +58,7 @@ def evaluateTokens(requestContext, tokens, replacements=None, pipedArg=None):
     return evaluateTokens(requestContext, tokens.template, arglist)
 
   if tokens.expression:
-    if tokens.expression.pipedCalls:
```

Comment:
Good catch on this! It is a subtle change, and was not in the pyparsing release notes (I have added it).

Reply:
Great. Thanks for the confirmation :)